### PR TITLE
Make open keyboard button more visible

### PR DESCRIPTION
**Pull request by dqnykamp**
This PR makes the open keyboard button be white with a black border to make it more visible, especially when it is in front of a dark background.

### CODE REVIEW

File: packages/virtual-keyboard/src/virtual-keyboard/keyboard-tray.css [hunk 1]
```diff
@@ -64,6 +63,8 @@ Tailwind base is purposely not included because we want all styles to be prefixe
         @apply border-b-black border-b;
         @apply hover:bg-slate-300 active:bg-slate-400;
         @apply w-12 h-[1.5rem] flex items-center justify-center;
+        background-color: white;
```
Comment (siefkenj) [inline packages/virtual-keyboard/src/virtual-keyboard/keyboard-tray.css:66]: This is fine. You can also do `bg-white` inside of an `@apply` block to do it tailwind style.

Comment (dqnykamp) [reply packages/virtual-keyboard/src/virtual-keyboard/keyboard-tray.css:66]: I was wondering about the `@apply`, but figured you'd let me know!

Comment (dqnykamp) [reply packages/virtual-keyboard/src/virtual-keyboard/keyboard-tray.css:66]: Is there a `border-width` `@apply` version I should use as well?

Comment (siefkenj) [reply packages/virtual-keyboard/src/virtual-keyboard/keyboard-tray.css:66]: https://tailwindcss.com/docs/border-style

**Comment by siefkenj** [comment]
Looks good.

**Pull request merged**